libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Converters/InterfaceConverter.cs

```diff
@@ -24,6 +25,9 @@ public class InterfaceConverter<T> : JsonConverter, IObservableConverter, IObser
         private readonly List<IJsonLoadObserver> observers = new List<IJsonLoadObserver>();
         private readonly SourceContext sourceContext;
 
+        private readonly ConcurrentDictionary<string, JToken> cachedRefJsons = new ConcurrentDictionary<string, JToken>();
```

Comment:
>JToken [](start = 54, length = 6)

I think the intent was to cache the typed object, not the JToken.  Doing the JToken means we will be creating new instance on every reference as we convert from JTokn= > Typed object.

Reply:
resovled